app/services/foreman/access_permissions.rb

```diff
@@ -32,13 +32,13 @@
                                              :"api/v1/auth_source_ldaps" => [:index, :show],
                                              :"api/v2/auth_source_ldaps" => [:index, :show]
     }
-    map.permission :create_authenticators,  {:auth_source_ldaps => [:new, :create],
+    map.permission :create_authenticators,  {:auth_source_ldaps => [:new, :create, :test_connection],
                                              :"api/v1/auth_source_ldaps" => [:create],
                                              :"api/v2/auth_source_ldaps" => [:create]
     }
     map.permission :edit_authenticators,    {:auth_source_ldaps => [:edit, :update],
```

Comment:
The edit permission ought to also give you the :test_connection AJAX permission, see also the compute resources bock.